borbskeeper/Utility/AlertManager.m

```diff
@@ -68,6 +75,14 @@ + (void) addCancelActionToAlert:(UIAlertController*)alert{
     [alert addAction:cancelAction];
 }
 
++ (void) addDeleteActionToAlert:(UIAlertController*)alert{
```

Comment:
```suggestion
+ (void) addDeleteActionToAlert:(UIAlertController*)alert {
```

Reply:
I think we need to change the parameters we receive here and add a block that will be executed when user taps on the this action.